assets/tests/components/map/controls/customControl.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { render } from "@testing-library/react"
+import "@testing-library/jest-dom"
+import React, { ReactNode } from "react"
+import { MapContainer, ZoomControl } from "react-leaflet"
+import { CustomControl } from "../../../../src/components/map/controls/customControl"
+import { FullscreenControl } from "../../../../src/components/map"
+
+const mapWrapper = ({ children }: { children: ReactNode }) => (
+  <MapContainer
+    center={[0, 0]}
+    zoom={0}
+    attributionControl={false}
+    zoomControl={false}
+  >
+    {children}
+  </MapContainer>
+)
+
+const customControlContents = () => {
+  return <button className="test-control-child-1">Click Me</button>
+}
+
+describe("CustomControls", () => {
+  test("when a position is specified, puts the control in the expected corner", () => {
+    const { container } = render(
+      <>
+        <CustomControl className="test-control" position="topright">
+          {customControlContents()}
+        </CustomControl>
+      </>,
+      {
+        wrapper: mapWrapper,
+      }
+    )
+
+    expect(
+      container.querySelector(".leaflet-control-container")
+    ).toMatchSnapshot()
```

Comment:
Snapshot testing seemed like the most straight-forward approach to me here in order to verify that the control ended up in exactly the expected place in the dom. Open to suggestions of other approaches though!

Reply:
I'm not sure if this is actually better, but would it be possible to do a more traditional test where you select the control and then look up the DOM tree using the `parentElement` property to verify that it's in the appropriate position?